Manual/Tutorial.tex

```diff
@@ -122,7 +122,7 @@ \subsection{Implementation}
 The first constructor creates the objects of $\Svec$.
 Here, the \CapPkg command
 \begin{itemize}
-  \item \texttt{ObjectifyObjectForCAPWithAttributes}
+  \item \texttt{CreateCapCategoryObjectWithAttributes}
 \end{itemize}
 creates a \GAP object that resides as an object in the category\footnote{
   Note that the usual creation of \GAP types is done automatically by \CapPkg.
```

Comment:
I don't think this note applies anymore -> just remove it.

Reply:
You only meant the footnote, right?

Reply:
Yes :-)